### PR TITLE
Implemented all styling changes on Add New Project page.

### PR DESCRIPTION
**Pull request by RensNooren**
## Description

Added the new designs of the "Project Add Page".

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [X] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my own code
-   [X] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I updated the changelog with an end-user readable description
-   [X] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Open "Add Project" page in the nav bar
2. Check the designs 

## Link to issue

Closes: #188

### CODE REVIEW

File: src/app/modules/project/add/source/source.component.html [hunk 1]
```diff
@@ -1,67 +1,59 @@
...
+<div class="row justify-content-center">
+  <div class="col-lg-6 col-md-8 col-sm-10 search">
+    <input type="search" placeholder="Paste a link to your project here" [formControl]="sourceUriInput">
+    <button (click)="onClickSubmitSourceUri()"><img src="assets/images/research.svg"></button>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-12 text-center info-box-add">
+    <p>You can use a link to your GitHub or GitLab repositories, for other projects you can use the option below.</p>
```
Comment (Brend-Smits) [inline src/app/modules/project/add/source/source.component.html:36]:
```suggestion
    <p>You can use a link to your GitHub or GitLab project, for other projects you can use the option below.</p>
```

File: src/app/modules/project/add/source/source.component.html [hunk 2]
```diff
@@ -1,67 +1,59 @@
...
+    <h3><a (click)="onClickAddProjectManual()">Add Project Manually</a></h3>
+    <p>Add your project manually if you don't have a project link</p>
+    <h1>or</h1>
+    <p>In the future you are able to upload projects by logging in to a specific source.<br>
+       For now, you can upload your projects with a link or by adding it manually.</p>
```
Comment (Brend-Smits) [inline src/app/modules/project/add/source/source.component.html:41]:
```suggestion
       For now, you can import your projects with a link or by adding it manually.</p>

```

File: src/app/modules/project/add/source/source.component.scss [hunk 3]
```diff
@@ -1,140 +1,168 @@
...
+
+  h3 {
+    margin-top: 10px;
+    margin-bottom: 15px;
+  }
+
+  margin-bottom: 75px;
+}
+
+.temporary-disabled {
```
Comment (MaxKorlaar) [inline src/app/modules/project/add/source/source.component.scss:166]:
```suggestion
.temporarily-disabled {

```

Comment (Brend-Smits) [reply src/app/modules/project/add/source/source.component.scss:166]: ty!

**Comment by Brend-Smits** [comment]
> I think SVGs would be better for the logos, because of scalability and file sizes

I agree, it's something we should take a look at in the future for sure.

**Pull request merged**